contrib/babelfishpg_tsql/src/multidb.c

```diff
@@ -689,8 +689,8 @@ rewrite_column_refs(ColumnRef *cref)
 			else
 			{
 				new_schema = makeString(get_physical_schema_name(cur_db, strVal(schema)));
-				list_delete_first(cref->fields);
-				lcons(new_schema, cref->fields);
+				cref->fields = list_delete_first(cref->fields);
+				cref->fields = lcons(new_schema, cref->fields);
```

Comment:
should get reviewed from whoever is owner of the code.

Reply:
This change has been suggested by the owner itself